### DISCUSSION TITLE
Termostat Heatit Z-TRM3: No entity created when the device is added

### DISCUSSION THREAD

**Discussion by cobch** [zwave-js/zwave-js#4996] [Q&A] [answered]
Hi everyone,
I am using Aeotec Z-Stick 7 with ZwaveJs addon on Home Assistant. I manage to add the device but not any entities is created and the node is described as not ready.
I have tried to heal and reinterview the device but it does not help.

Any ideas how to fix that?
<img width="760" alt="Screenshot 2022-09-02 at 09 20 15" src="https://user-images.githubusercontent.com/50397676/188081174-6e88d2ef-9e84-4acc-9ea4-0fcfc596f144.png">

**Comment by AlCalzone** [comment, marked as answer]
Please open a [bug report](https://github.com/zwave-js/node-zwave-js/issues/new?assignees=&labels=&template=bug_report.yml) and provide the requested information, especially the driver log.
